backend/app/adapter/sqldb/short_link.go

```diff
@@ -18,6 +18,34 @@ type ShortLinkSql struct {
 	db *sql.DB
 }
 
+// UpdateTwitterTags updates Twitter meta tags for a given short link.
+func (s *ShortLinkSql) UpdateTwitterTags(alias string, twitterTags metatag.Twitter) (entity.ShortLink, error) {
```

Comment:
Can the implementation of UpdateTwitterTags be extracted out into a private function to be used also for UpdateOpenGraphTags?

Reply:
@arberiii I think we are conceptually different. Prefer keeping them separate so that we can make independent changes in the future.

Reply:
How @arberiii, there Update statements are different